src/components/common/fragments/AddOrEdit.tsx

```diff
@@ -1,21 +1,17 @@
-import React, { FunctionComponent } from "react";
-import { DavitAddButton } from './buttons/DavitAddButton';
+import React, {FunctionComponent} from "react";
+import {DavitAddButton} from './buttons/DavitAddButton';
 
 interface AddOrEditProps {
     addCallBack: () => void;
-    label: string;
     dropDown: JSX.Element;
```

Comment:
bitte schau mal ob die bestimmen kannst das das hier ein dropdown sein muss (https://stackoverflow.com/questions/44475309/how-do-i-restrict-the-type-of-react-children-in-typescript-using-the-newly-adde)